types/bunyan-format/bunyan-format-tests.ts

```diff
@@ -0,0 +1,8 @@
+import bunyan = require('bunyan');
```

Comment:
This doesn't seem to show the two libraries interacting in any way?

Reply:
I simplified test for this module and now it doesn't use bunyan itself.